cofmpy/helpers/extract_fmu.py

```diff
@@ -51,18 +51,8 @@ def extract_fmu_info(fmu_path):
         console.print(f"[red]❌ Error: FMU file '{fmu_path}' not found.[/red]")
         return
 
-    # Extract FMU content
-    unpacked_fmu_dir = extract(fmu_path)
-    model_desc = read_model_description(fmu_path)
-
-    # Prepare table
-    table = Table(
-        title=f"📜 FMU Information: {os.path.basename(fmu_path)}", show_lines=True
-    )
-    table.add_column("Category", style="cyan", no_wrap=True)
-    table.add_column("Variable", style="magenta")
-    table.add_column("Type", style="green")
-    table.add_column("Start Value", style="white")
+    unpacked_fmu_dir = fmpy.extract(fmu_path)
```

Comment:
I know it's not new code from this PR, but why do we need to unpack the FMU zip file? It seems to be never used in the function? If it's not useful, I suggest to remove the `unpacked_fmu_dir` variable in the functions here